CHANGELOG.md

```diff
@@ -51,6 +51,7 @@ Bottom level categories:
 
 #### General
 
+- Implement `Hash` for `DepthStencilState` and `DepthBiasState` in `wgpu-types`.
```

Comment:
```suggestion
- Implement `Hash` for `DepthStencilState` and `DepthBiasState`.
```

We don't need to mention wgt as it's not part of our public interface, we re-export everything under the `wgpu` banner.

Reply:
Okay, now I know.